video/out/opengl/video.c

```diff
@@ -1688,6 +1688,15 @@ static bool eval_szexpr(struct gl_video *p, struct img_tex tex,
             struct bstr name = expr[i].val.varname;
             struct img_tex var_tex;
 
+            // The size of OUTPUT is determined. It could be useful for certain
+            // user shaders to skip passes.
+            if (bstr_equals0(name, "OUTPUT")) {
+                stack[idx++] = (expr[i].tag == SZEXP_VAR_W) ?
+                               ((float)p->dst_rect.x1 - (float)p->dst_rect.x0) :
+                               ((float)p->dst_rect.y1 - (float)p->dst_rect.y0);
```

Comment:
The float casts are unnecessary

(For clarity I would probably define these as `int vp_w, vp_h` first, that would allow you to avoid the multi-line ternary)